doc-content/enterprise-only/springboot/bus-app-security-preauthenticate-proc.adoc

```diff
@@ -0,0 +1,102 @@
+[id='bus-app-security-preauthenticate-proc_{context}']
+
+= Using Spring Security with preauthenication
+If you disable Spring Security authentication by using the `PermitAll` method, any user can log in to the application, but users will have limited access and functionality. However, you can preauthenticate a user, for example a designated service account, so a group of users can use the same login but have all of the permissions that they require. That way, you do not need to create credentials for each user.
+
```

Comment:
Possible change to:

and have all of the permissions that they require without creating credentials for each user.